mcp/mcp-bundle-api/src/main/resources/META-INF/smithy/mcpbundle.smithy

```diff
@@ -17,3 +18,9 @@ structure BundleMetadata {
 
     version: String
 }
+
+structure CodeRepoBundle {
+    // The URL of this MCP server's code repository
```

Comment:
If this was meant to be a [documentation comment](https://smithy.io/2.0/spec/idl.html#documentation-comment), you need `///`